test/check-application

```diff
@@ -1003,6 +1032,30 @@ class TestFiles(testlib.MachineCase):
         b.key("ArrowLeft")
         b.wait_visible("[data-item='file0'].row-selected")
 
+        # Go up and down in directory hierarchy
+        b.click("[data-item='testdir']")
+        b.key("ArrowDown", modifiers=1)
+        b.wait_text(".breadcrumb-button:nth-of-type(5)", "testdir")
+        b.key("ArrowUp", modifiers=1)
+        b.wait_visible("[data-item='testdir']")
+
+        # Manually edit path
+        b.key("L")
+        b.input_text("/home/admin/anotherdir")
+        b.key("Enter")
+        b.wait_text(".breadcrumb-button:nth-of-type(5)", "anotherdir")
+
+        # Go back in history twice
+        # b.key("ArrowLeft", repeat=2, modifiers=1)
```

Comment:
@jelly do you have any idea what might be happening here? the `b.key(...)` doesn't seem to work when running this test in chromium but everything works fine in firefox